spec/index.bs

```diff
@@ -7349,13 +7349,29 @@ interface GPUSwapChain {
 GPUSwapChain includes GPUObjectBase;
 </script>
 
-In the "update the rendering [of the] `Document`" step of the "Update the rendering" HTML processing
-model, the contents of the {{GPUTexture}} most recently returned by
-{{GPUSwapChain/getCurrentTexture()}} are used to update the rendering for the `canvas`, and it is as
-if {{GPUTexture/destroy()}} were called on it (making it unusable elsewhere in WebGPU).
+{{GPUSwapChain}} has the following internal slots:
 
-Before this drawing buffer is presented for compositing, the implementation shall ensure that all
-rendering operations have been flushed to the drawing buffer.
+<dl dfn-type=attribute dfn-for="GPUSwapChain">
+    : <dfn>\[[format]]</dfn> of type {{GPUTextureFormat}}.
+    ::
+        The format of the textures returned by this swap chain.
+
+    : <dfn>\[[textureUsage]]</dfn> of type {{GPUTextureUsage}}.
+    ::
+        The usage of the textures returned by this swap chain.
```

Comment:
For these two members I think we should move to having a `[[descriptor]]` instead, instead of dissolving the descriptor out.